pkg/ebpf/cached_conntrack.go

```diff
@@ -0,0 +1,164 @@
+// +build linux_bpf
+
+package ebpf
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/DataDog/datadog-agent/pkg/network/netlink"
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	ct "github.com/florianl/go-conntrack"
+	"github.com/golang/groupcache/lru"
+	"golang.org/x/sys/unix"
+)
+
+type refCountedConntrack struct {
+	ctrk  netlink.Conntrack
```

Comment:
nit (feel free to ignore): I think embedding this would be a bit more idiomatic here